src/legacy/core_plugins/kibana/public/discover/np_ready/angular/helpers/_get_point.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Aspect, Table, Row } from './point_series';
+
+type RowValue = number | 'NaN';
+interface Raw {
+  table: Table;
+  column: number;
+  row: number;
+  value: RowValue;
+}
+export interface Point {
+  x: RowValue;
+  y: RowValue;
+  extraMetrics: [];
+  xRaw: Raw;
+  yRaw: Raw;
+  tableRaw?: {
+    table: Table;
+    column: number;
+    row: number;
+    value: number;
+    title: string;
+  };
+  parent: null;
+  series?: string;
```

Comment:
Same for the `series` concept - as there is always just a single series, we can just pass it through directly as `values` or something and getting rid of the nested structure. All of the things put together here are just used in a single place - https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/kibana/public/discover/np_ready/angular/directives/histogram.tsx#L208

If it's not used there, we can just throw it out.